bc_obps/service/report_service.py

```diff
@@ -121,11 +119,9 @@ def get_facility_report_by_version_and_id(
 
     @classmethod
     def get_activity_ids_for_facility(cls, facility_id: int) -> List[int]:
-        try:
-            facility = FacilityReport.objects.get(id=facility_id)
-            return list(facility.activities.values_list('id', flat=True))
-        except FacilityReport.DoesNotExist:
-            return []
+        # Fetch the facility and return the list of activity IDs
+        facility = FacilityReport.objects.get(id=facility_id)
```

Comment:
This is a little confusing. Do we mean to access it by facility ID? or by facility_report id? 
The current call will always return `None` since facility_report_id is of type int and facility_id is a UUID in our system

Reply:
This was changed as part of the fixes I made in #2223